### PR TITLE
[#633] update python package version fields

### LINKED ISSUE #633: Update Python's package version fields in update_version_id script
- fix build user name
- remove build id

### PR DESCRIPTION
**Pull request by botsman**
This closes #633

- removed build_id from `update_version_id`
- fix passing `BUILD_USER` in Jenkinsfile

### CODE REVIEW

**Review by kirillmakhonin — DISMISSED** [review]

**Comment by botsman** [comment]
bot, test all

**Comment by botsman** [comment]
bot, test all

**Comment by botsman** [comment]
bot, test all

**Comment by botsman** [comment]
bot, test all

**Comment by botsman** [comment]
bot, test all

**Comment by botsman** [comment]
bot, test all

**Comment by botsman** [comment]
bot, test infra

**Comment by botsman** [comment]
bot, test infra

**Comment by botsman** [comment]
bot, test infra

**Comment by botsman** [comment]
bot, test infra

**Comment by botsman** [comment]
bot, test infra

**Review by mcdoker18 — DISMISSED** [review]

**Review by kirillmakhonin — DISMISSED** [review]

**Comment by mcdoker18** [comment]
bot, test infra

File: Jenkinsfile [hunk 1]
```diff
@@ -75,22 +75,30 @@ pipeline {
...
 
-                    Globals.dockerLabels = "--label git_revision=${Globals.rootCommit} --label build_id=${env.BUILD_NUMBER} --label build_user=${env.BUILD_USER} --label build_date=${buildDate}"
+                    wrap([$class: 'BuildUser']) {
+                        if (binding.hasVariable('BUILD_USER ')) {
+                            BUILD_USER = "${BUILD_USER}"
+                        } else {
+                            BUILD_USER = null
+                        }
+                    }
+
```
Comment (alexander-semenets) [inline Jenkinsfile:81]: Can we make it a bit smoother?
smth like this for example:
BUILD_USER = (binding.hasVariable('BUILD_USER ') ? "${BUILD_USER}" : 'null'

Comment (botsman) [reply Jenkinsfile:81]: Yeah, sure

**Comment by botsman** [comment]
bot, test infra

**Review by mcdoker18 — APPROVED** [review]

**Comment by mcdoker18** [comment]
bot, test infra

**Review by kirillmakhonin — APPROVED** [review]

**Review by alexander-semenets — APPROVED** [review]

**Pull request merged**